### ISSUE TITLE
Something wrong in OpenAI's Large-v2 model?

### ISSUE THREAD

**Issue by ggerganov** [ggerganov/whisper.cpp#674]
So, I haven't looked in details, but I suspect there might be something wrong in the new `large` model released by OpenAI.
Keep in mind this is very anecdotal evidence atm, so I might be completely wrong.

Running the `main` example with enabled color coding for the token probabilities `-pc` we normally get the following results:

- `base.en`

![image](https://user-images.githubusercontent.com/1991296/228167578-43f15218-da45-41e1-9c2d-f6dc95e38887.png)

- `small.en`

![image](https://user-images.githubusercontent.com/1991296/228167909-5a64c029-15e8-42d5-a97b-ad5ce61c3bf1.png)

- `small`

![image](https://user-images.githubusercontent.com/1991296/228168161-21771e9a-ff40-4e6b-8c43-a946febc648b.png)

- `medium`

![image](https://user-images.githubusercontent.com/1991296/228168508-0aed6873-f5ab-401c-8521-8de8aaf6a4a3.png)

However, this is what the color coding look like when using the new `large` model (i.e. v2):

- `large`

![image](https://user-images.githubusercontent.com/1991296/228168723-5c4e2311-91a8-49e0-8a53-8ea75a613446.png)

As a comparison, this is the same run, but using the old `large` model - i.e. v1:

- `large-v1`

![image](https://user-images.githubusercontent.com/1991296/228168909-4b18af92-f5b7-4c55-8b7c-7995a3b42bac.png)

So somehow the logits with `v2` seem to be all over the place which is not observed for any of the other models.
Still, I need to double check all these observations, but I think there is something not quite right with `large-v2`
